src/Relude/Monad/Either.hs

```diff
@@ -112,46 +112,86 @@ maybeToLeft :: r -> Maybe l -> Either l r
 maybeToLeft r = maybe (Right r) Left
 {-# INLINE maybeToLeft #-}
 
--- | Applies given action to 'Either' content if 'Left' is given and returns
+{- | Applies given action to 'Either' content if 'Left' is given and returns
 -- the result. In case of 'Right' the default value will be returned.
+
+>>> whenLeft "bar" (Left "foo") (\a -> [a])
+["foo"]
+
+>>> whenLeft "bar" (Right "foo") (\a -> [a])
```

Comment:
To me, this makes the example more clear. To show that value under `Right` can have a different type. So you immediately see the behaviour
```suggestion
>>> whenLeft "bar" (Right 42) (\a -> [a])
```